test/wca_live_web/controllers/competition_controller_test.exs

```diff
@@ -17,6 +17,49 @@ defmodule WcaLiveWeb.CompetitionControllerTest do
       assert %{"formatVersion" => "1.0", "id" => "WC2019"} = body
     end
 
+    test "returns results for a competition", %{conn: conn} do
```

Comment:
Btw. currently this is in the `"show_wcif"` describe block. Let's move it to its own describe block below :)